packages/playwright-test/src/matchers/matchers.ts

```diff
@@ -164,6 +164,23 @@ export function toHaveClass(
   }
 }
 
+export function toContainClass(
+  this: ReturnType<Expect['getState']>,
+  locator: LocatorEx,
+  expected: string | string[],
+  options?: { timeout?: number },
```

Comment:
`ignoreCase` right away?

Reply:
Not sure if this would worth doing it. But implementation wise it would mean, that I should use `expectedText` instead of `expectedValue`  to pass the `ignoreCase` value over like we do in the other places.